FV3/atmos_cubed_sphere/model/fv_mapz.F90

```diff
@@ -1389,7 +1395,9 @@ subroutine map_scalar( km,   pe1,    q1,   qs,           &
 
 ! Compute vertical subgrid distribution
    if ( kord >7 ) then
-        call scalar_profile( qs, q4, dp1, km, i1, i2, iv, kord, q_min )
+     
+      call scalar_profile( qs, q4, dp1, km, i1, i2, iv, kord, q_min )
+     
```

Comment:
Would be nice if there was no diff here (only changes are whitespace).

Reply:
thanks for pointing that out -- I agree, we should try to avoid needless whitespace changes